upa/contracts/UpaVerifier.sol

```diff
@@ -934,7 +932,7 @@ contract UpaVerifier is
 
     /// Withdraws the worker's balance in the `feeModel` contract,
     /// including the collateral.
-    function withdrawAggregatorBalance() external {
+    function withdrawAggregatorBalance() external onlyWorker {
```

Comment:
do we want this one to be `onlyWorker` or `onlyOwner`? I'd argue `onlyOwner` makes more sense, because the worker account is the weakest link in the chain and we won't be calling this function unless we're moving to another contract in a non-upgradeable way.

Reply:
I can see an argument for both.  Arguably it's collateral owned by the worker, so logically it should be him that initiates the withdrawal.  However given that we don't have any conditions for withdrawal owner may make more sense.  The worker needs permission from the owner to retire.  It's also the owner who assigns a new aggregator.

Reply:
Given that the owner is supposed to assign/retire the worker, I've changed to `onlyOwner` here.